grc/core/platform.py

```diff
@@ -341,6 +340,18 @@ def parse_flow_graph(self, filename):
             from ..converter.flow_graph import from_xml
             data = from_xml(filename)
 
+        file_format = data.get('metadata', {}).get('file_format')
+        if file_format is None:
+            Messages.send(
+                '>>> WARNING: Flow graph does not contain a file format version!\n')
+        elif file_format == 0:
+            Messages.send(
+                '>>> WARNING: Flow graph format is version 0 (legacy) and will'
+                ' be converted to version 1 or higher upon saving!\n')
+        elif file_format > Constants.FLOW_GRAPH_FILE_FORMAT_VERSION:
+            raise RuntimeError(
```

Comment:
FYI throwing here is the same behaviour as with flowgraphs that don't pass schema checking, so I thought that was appropriate.